src/components/Banner/Banner.tsx

```diff
@@ -63,14 +63,14 @@ export const Banner = ({
   variant,
   closeButtonText = 'Close',
   ...other
-}) => {
+}: Props) => {
   const componentClassName = clsx(
     styles['banner'],
     variant === 'success' && styles['banner--success'],
     variant === 'warning' && styles['banner--warning'],
     variant === 'error' && styles['banner--error'],
     variant === 'brand' && styles['banner--brand'],
-    variant === 'vertical' && styles['banner--vertical'],
+    orientation === 'vertical' && styles['banner--vertical'],
```

Comment:
FYI @csnizik I think this was in the "prop clean-up ticket" but I fixed it here since I was already in the file